CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 * **core + css:** De HTML/CSS Storybook en Core Storybook zijn gefuseerd. ([#1659](https://github.com/dso-toolkit/dso-toolkit/issues/1659))
 * **core:** Autosuggest: `aria-owns` op de input vervangen met `aria-controls` ([#1716](https://github.com/dso-toolkit/dso-toolkit/issues/1716))
 * **css + dso-toolkit:** Panel verhuizen naar @dso-toolkit/css ([#1476](https://github.com/dso-toolkit/dso-toolkit/issues/1476))
+* **core + css + sources + storybook** Viewer grid: gekozen filters in lijst, label verwijderknopjes uitbreiden ([#1712](https://github.com/dso-toolkit/dso-toolkit/issues/1712))
```

Comment:
Laten we wat meer zorg dragen voor de CHANGELOG entries. Desnoods moeten we de titel van de issues aanpassen naar wat we gemaakt hebben, maar wat hier nu staat dekt de lading niet.

Dat geldt voor dat andere PR ook, wil je die meteen bijwerken?

Reply:
Mee eens, ik zal eens wat passends gaan verzinnen